en/community/weblogs/index.md

```diff
@@ -21,6 +21,9 @@ Ruby code, describing new techniques, or speculating on Ruby’s future.
   and other Ruby resources.
 * [**Rubyland**][rubyland] aggregates news and blog posts about Ruby
   from RSS feeds.
+* [**Rails at Scale**][rails-at-scale] contains posts about the work
+  being done around both Ruby and Rails by the Ruby and Rails Infrastructure
+  team at Shopify.
```

Comment:
Thank you for your contribution! 

The content from the site is absolutely valuable and relevant. However, my opinion is that Ruby official site should maintain neutrality and avoid promoting specific companies. I suggest rephrasing the description to emphasize the content’s relevance to Ruby without referencing the team/company behind it (but definitely appreciate all the work the team/company are doing 💚).

Reply:
Also should this be under `Blogs of Note`?

Reply:
Yeah, that's fair. I tried rephrasing it, let me know what you think.

> Also should this be under Blogs of Note?

I moved it there. Is `Mining for blogs` supposed to be a list of aggregators where you can discover blogs? I guess I didn't understand the title at first glance.